cleanup & formatting for code samples.

subhamkumarr opened the pull request:
Resolves: https://github.com/MetaMask/metamask-docs/issues/1122

This pull request attempts to ensure that the code samples in the docs follow some consistent style guidelines. To accomplish this, the code samples were formatted using the Prettier formatter plugin and the following rules:

Double quotes preferred over single quotes
4-space indentation preferred over 2-space
Trailing commas preferred over no trailing commas
No metadata preferred over metadata

ziad-saab reviewed (changes requested):
@subhamkumarr thank you for taking this on! Here's the approach I'd like you to take instead:

1. Install [`eslint-plugin-markdown`](https://github.com/eslint/eslint-plugin-markdown)
2. Create a Prettier config
3. Make sure [ESLint and Prettier play well with each other](https://github.com/prettier/eslint-plugin-prettier?tab=readme-ov-file#installation)
4. Run auto-fix and make sure everything works

subhamkumarr commented:
@ziad-saab Firsly I ran this in the terminal `npm install --save-dev eslint-plugin-markdown` then 
 i have created an ` .prettierrc` file and added this code
```
{
  "semi": false,
  "singleQuote": false,
  "quoteProps": "consistent",
  "tabWidth": 4,
  "useTabs": false,
  "trailingComma": "all"
}

```
But when I am running this command `npx eslint --fix .`  I am getting this error in the console
```
Oops! Something went wrong! :(

ESLint: 8.56.0

Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'C:\Users\Subham Kumar\Desktop\metamask-docs\node_modules\eslint-plugin-markdown\configs\recommended' imported from C:\Users\Subham Kumar\Desktop\metamask-docs\eslint.config.js
    at finalizeResolution (node:internal/modules/esm/resolve:264:11)
    at moduleResolve (node:internal/modules/esm/resolve:917:10)
    at defaultResolve (node:internal/modules/esm/resolve:1130:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
    at link (node:internal/modules/esm/module_job:84:36)
```

I have also added "type": "module" in package.json file
